applications_and_projects/operators/osdk-ansible.adoc

```diff
@@ -0,0 +1,53 @@
+:relfileprefix: ../
+[id='osdk-ansible']
+= Creating Ansible-based Operators
+include::modules/common-attributes.adoc[]
+:context: osdk-ansible
+
+toc::[]
+
+{nbsp} +
+This guide outlines Ansible support in the Operator SDK and walks Operator
+authors through examples building and running Ansible-based Operators with the
+`operator-sdk` CLI tool that use Ansible playbooks and modules.
+
+include::modules/osdk-ansible-support.adoc[leveloffset=+1]
+include::modules/osdk-ansible-custom-resource-files.adoc[leveloffset=+2]
+include::modules/osdk-ansible-watches-file.adoc[leveloffset=+2]
+include::modules/osdk-ansible-extra-variables.adoc[leveloffset=+2]
+include::modules/osdk-ansible-runner-directory.adoc[leveloffset=+2]
+
+include::modules/osdk-installing-cli.adoc[leveloffset=+1]
+include::modules/osdk-building-ansible-operator.adoc[leveloffset=+1]
+
+[id='osdk-ansible-k8s-module']
+== Managing application lifecycle using the k8s Ansible module
+
+To manage the lifecycle of your application on Kubernetes using Ansible, you
+can use the link:https://docs.ansible.com/ansible/2.7/modules/k8s_module.html[`k8s` Ansible module].
+This Ansible module allows a developer to either leverage their existing
+Kubernetes resource files (written in YAML) or express the lifecycle management
+in native Ansible.
+
+One of the biggest benefits of using Ansible in conjunction with existing
+Kubernetes resource files is the ability to use Jinja templating so that you can
+customize resources with the simplicity of a few variables in Ansible.
+
+This section goes into more detail on usage of the `k8s` Ansible, which was
+touched on in the previous procedure. To get started, install the module on your
```

Comment:
Just wondering if you should say "previous procedure" here, since the procedure is in a module and what if someone added another module in between that and this text?

Reply:
I try to avoid "previous/following" like this, but thought maybe I was safe being at the assembly level where there's more control. But, on second reading, it's superfluous, plus I accidentally'd at word ("module"). Changed!